src/gdcdictionary/schemas/_terms.yaml

```diff
@@ -1212,8 +1212,8 @@ columnar_mucosa_present:
 comorbidities:
   common:
     description: >-
-      The text term used to describe a comorbidity disease, which coexists with the
-      patient's malignant disease.
+      The text term used to describe a patient with one or more diseases or conditions,
+      which coexists with the patient's malignant disease at the same time.
```

Comment:
I seriously have no idea if "coexists" or "coexist" should be used here.

Reply:
> I seriously have no idea if "coexists" or "coexist" should be used here.

me too. seems like "one or more" can take the singular or plural, but then the "which" can refer to either, so...

Reply:
Updated to "coexist", I think it flows better and makes more grammatical sense given diseases/conditions are plural